### PR TITLE
Fix timezone-related test and model issues.

### PR DESCRIPTION
**Pull request by umgelurgel**
Tests no longer generate naive datetime warnings.

### CODE REVIEW

File: assopy/models.py [hunk 1]
```diff
@@ -1095,7 +1095,7 @@ def save(self, *args, **kwargs):
             except Refund.DoesNotExist:
                 pass
         if self.status in ('rejected', 'refunded') and self.status != old:
-            self.done = datetime.now()
+            self.done = now()
```
Comment (artcz) [inline assopy/models.py:1098]: :+1: however `timezone.now()` sounds less ambiguous.

File: tests/test_cfp.py [hunk 2]
```diff
@@ -275,8 +275,8 @@ def test_ignores_new_tags(self):
         assert Tag.objects.count() == 0
 
     def test_ignores_new_tags_keeping_predefined_ones(self):
-        ConferenceTag.objects.create(name='django')
-        ConferenceTag.objects.create(name='love')
+        ConferenceTag.objects.create(name=u'django')
+        ConferenceTag.objects.create(name=u'love')
```
Comment (artcz) [inline tests/test_cfp.py:279]: not sure if necessary

**Review by artcz — APPROVED** [review]
LGTM

File: tests/test_cfp.py (hunk 2 above)
Comment (umgelurgel) [reply tests/test_cfp.py:279]: I think this was generating a warning, hence the change.

**Pull request merged**